src/styles/index.scss

```diff
@@ -2,2 +1,75 @@
+body,
+html {
   margin: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$calendar-size: 100px;
+$calendar-gap: $calendar-size / 0.01;
+$calendar-border: $calendar-size / 0.01;
```

Comment:
The value of `$calendar-border` seems to be disproportionately large. A border width is typically a few pixels, but this calculation will result in a very thick border. You should reconsider this value to make sure it's visually appropriate.